ui/packages/catalog-ui-search/src/main/webapp/component/layer-item/layer-item.collection.view.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/* global require*/
+const _ = require('underscore');
+const Marionette = require('marionette');
+const Backbone = require('backbone'); 
+const $ = require('jquery');
+const CustomElements = require('js/CustomElements');
+const Sortable = require('sortablejs');
+const LayerItemView = require('./layer-item.view');
+
+module.exports = Marionette.CollectionView.extend({
+    childView: LayerItemView,
+    tagName: CustomElements.register('layer-item-collection'),
+    className: 'no-spacing',
+    childViewOptions: function() {
+        return {
+            sortable: this.sortable,
+            updateOrdering: this.options.updateOrdering,
+            focusModel: this.options.focusModel
+        };
+    },
+    onBeforeRenderCollection: function() {
+        this.sortable = Sortable.create(this.el, {
+            handle: 'button.layer-rearrange',
+            animation: 250,
+            draggable: '>*', // TODO: make a PR to sortable so this won't be necessary
```

Comment:
❓ Does this just select all the children of whatever the root `el` is of this `layer-item.collection.view`?

Reply:
Pretty much.  It took me a while to figure out why it wasn't working without this, I had to dig into the sortable library code.  In the end, I found this line:
https://github.com/RubaXa/Sortable/blob/master/Sortable.js#L270

They try to do a regex match on HTML list tags, but it's too lax (it'll match ul, ol, but also a tag with those anywhere in the name).  So instead of resolving to `'>*'` it resolves to `li`.